pages/checkout/access.tsx

```diff
@@ -78,31 +84,24 @@ const CheckoutAccessView = ({ assetId }: AssetIdProp) => {
                 <Ingress>
                   {intl.formatMessage({ id: "checkout.access.ingress" })}
                 </Ingress>
-
-                <ExampleContainer>
-                  <Typography variant="caption">
-                    {intl.formatMessage({ id: "checkout.access.exampleLabel" })}
-                  </Typography>
-                  <Typography variant="caption">
-                    {intl.formatMessage({ id: "checkout.access.exampleBody" })}
-                  </Typography>
-                </ExampleContainer>
-
-                <TextFieldContainer>
-                  <TextField
-                    multiline
-                    id="description"
-                    label={intl.formatMessage({ id: "checkout.access.descriptionInput.label" })}
-                    placeholder={intl.formatMessage({ id: "checkout.access.descriptionInput.placeholder" })}
-                    onChange={onDescriptionChange}
-                    value={checkoutData?.access?.description}
-                  />
-                </TextFieldContainer>
-
+                <TextField
+                  multiline
+                  id="description"
+                  label={intl.formatMessage({ id: "checkout.access.descriptionInput.label" }, { maxLength: MAX_LENGTH })}
+                  placeholder={intl.formatMessage({ id: "checkout.access.descriptionInput.placeholder" })}
+                  onChange={onDescriptionChange}
+                  value={description}
+                  rows={4}
+                  variant={error ? "error" : "default"}
+                  maxLength={MAX_LENGTH}
+                  meta={`${description && description.length}`}
```

Comment:
renders `"undefined"` until the user starts typing

Reply:
Outsmarted myself when I tried to fool EDS 😂